config/initializers/simple_form.rb

```diff
@@ -10,7 +10,7 @@
     b.use :input, class: 'field'
   end
 
-  config.wrappers :vertical_form, tag: 'div', class: 'mb2', error_class: 'has-error' do |b|
+  config.wrappers :vertical_form, tag: 'div', class: 'mb3', error_class: 'has-error' do |b|
```

Comment:
decision reached -- using mb3 as default and overriding to mb2 where necessary (i.e., idv -- `= f.input :foo, wrapper_html: { class: 'mb2' }`